src/main/java/io/apicurio/datamodels/core/validation/rules/required/AasMissingServerProtocolRule.java

```diff
@@ -0,0 +1,21 @@
+package io.apicurio.datamodels.core.validation.rules.required;
+
+import io.apicurio.datamodels.core.Constants;
+import io.apicurio.datamodels.core.models.common.Server;
+import io.apicurio.datamodels.core.validation.ValidationRuleMetaData;
+
+public class AasMissingServerProtocolRule extends RequiredPropertyValidationRule {
+
+	public AasMissingServerProtocolRule(ValidationRuleMetaData ruleInfo) {
+		super(ruleInfo);
```

Comment:
You have some java files that are using tabs instead of spaces as the formatting.  Could you go through all new java files you've created and make sure they are formatted according to that formatter.xml settings I mentioned before?

Reply:
👍 yep, I had added it, just forgot to format the file..